lib/spark_api/models/search_template/quick_search.rb

```diff
@@ -5,7 +5,12 @@ class QuickSearch < Base
       include Concerns::Savable,
               Concerns::Destroyable
 
-      self.element_name="searchtemplates/quicksearches"
+      self.element_name = "searchtemplates/quicksearches"
+
+      def fields(args = {})
+        arguments = {:_expand => "Fields"}.merge(args)
```

Comment:
Sort of. It's good that you memoized them like that, but consider this case: 

```ruby
@search = QuickSearch.find('1234', :_expand => 'Fields')
# GET  /v1/searchtemplates/quicksearches/1234?_expand=Fields

puts @search.fields
# GET /v1/searchtemplates/quicksearches/1234?expand=Fields
```

Wouldn't calling @search.fields for the first time on any QuickSearch instance result in a second API call, whether the original API call included them or not?